src/blocks/stopwatch.scss

```diff
@@ -0,0 +1,48 @@
+.stopwatch {
+  position: relative;
+  @include circleSize($size-stop-watch);
+
+  background: $bg-stop-watch;
+  
+  &__minut::before {
+    content: "";
+    @include positArrow($width-minut);
+    width: $width-minut;
+    height: $hight-minut;
+
+    background: $bg-minut;
+    border-radius: $radius-arrow;
+    
+    animation: $anime-minut;
+    transform-origin: bottom;
```

Comment:
Maybe a mixing for these? You repeat the exact same properties in two places.

Reply:
yep, thanks i missed it ))